packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,235 @@
+import { darken, fade, lighten, makeStyles } from '@material-ui/core/styles';
+
+export const getStyles = makeStyles((theme) => ({
+  'MuiDataGrid-root': {
+    lineHeight: theme.typography.pxToRem(24),
+    'box-sizing': 'border-box',
+    position: 'relative',
+    'font-family': theme.typography.fontFamily,
+    'letter-spacing': theme.typography.body2,
+    border: '1px solid #bdc3c7',
+    'border-radius': theme.shape.borderRadius,
+    outline: 'none',
+    display: 'flex',
+    flex: 1,
+    'flex-direction': 'column',
+
+    '& *': {
+      'box-sizing': 'border-box',
+    },
+
+    '& .main-grid-container': {
+      position: 'relative',
+      'flex-grow': 1,
+      'flex-direction': 'column',
+      display: 'flex',
+    },
+
+    '& .watermark': {
```

Comment:
What about we use inline style for the watermark? So that:

- it's harder to override (need !important)
- we have the styles even with the unstyled version of the grid.